### DISCUSSION TITLE
Welcome to finamp Discussions!

### DISCUSSION THREAD

**Discussion by UnicornsOnLSD** [UnicornsOnLSD/finamp#4] [General]
GitHub told me to enable this so I did :)

If I ever release this app, support requests and suggestions should probably go here

**Comment by dpraul** [comment]
Really excited about this! Thanks for working on it.

I saw you already added an entry to jellyfin-docs - you can add it to https://github.com/jellyfin/jellyfin-blog/blob/master/content/clients/_index.md to get it added to https://jellyfin.org/clients/ :)

**Reply by UnicornsOnLSD** [reply]
Just submitted a pull request :)
